bundler/lib/bundler/cli/gem.rb

```diff
@@ -441,12 +441,8 @@ def open_editor(editor, file)
       thor.run(%(#{editor} "#{file}"))
     end
 
-    def rust_builder_required_rubygems_version
-      "3.3.11"
-    end
-
```

Comment:
The new gemspec template uses this method to set a lower bound on the version minimum version of rubygems, so we should probably keep it?